stylesheets/app.scss

```diff
@@ -2085,3 +2085,12 @@ h2.home-title {
     }
   }
 }
+
+// Hacking for drupal
```

Comment:
It's in the commit message for this change

On Thursday, August 25, 2016, Rob Blake <notifications@github.com> wrote:

> In stylesheets/app.scss
> <https://github.com/redhat-developer/developers.redhat.com/pull/1385#discussion_r76225227>
> :
>
> > @@ -2085,3 +2085,12 @@ h2.home-title {
> >      }
> >    }
> >  }
> > +
> > +// Hacking for drupal
>
> @LightGuard <https://github.com/LightGuard> Could you please document in
> the comment here what this works around in Drupal so that if we ever want
> to remove it in the future, we can work out what it is doing.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/1385/files/72bfc419a5e0c11fc8ee3fe6823d185c3ea84a56#r76225227>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAB1E175-V7hHNSY9uW4XRiRAMTjaV4aks5qjXsjgaJpZM4Jshjg>
> .
>


-- 
Sent from Gmail Mobile